setup.cfg

```diff
@@ -1,3 +1,41 @@
+[metadata]
+name = dvo-extractor
+author = Red Hat Inc.
+description-file = README.md
+license = Apache 2.0
+long_description_content_type = text/markdown
+home-page = https://github.com/RedHatInsights/dvo-extractor
+classifier =
+    Intended Audience :: Information Technology
+    Intended Audience :: System Administrators
+    Operating System :: POSIX :: Linux
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
+
+[options]
+zip_safe = False
+packages = find:
+install_requires =
+    app-common-python
+    ccx-messaging>=3.4.2
+    insights-core-messaging
+setup_requires =
+    setuptools
+    setuptools_scm
+    wheel
+
+[options.packages.find]
+exclude =
+    test*
+
+[options.entry_points]
+console_scripts =
+    dvo-extractor = dvo_extractor.command_line:insights_dvo_extractor
+
 [pycodestyle]
 max-line-length = 100
 
```

Comment:
we could, but it would not be the more PEP'py way =)

I updated the files in order to remove the requirements.txt file completely though. Thanks for pointing it out!